ui/lib/kv/addon/components/page/secret/details.hbs

```diff
@@ -27,41 +27,41 @@
   </:toolbarActions>
 </KvPageHeader>
 
-{{#if this.emptyState}}
-  <EmptyState @title={{this.emptyState.title}} @message={{this.emptyState.message}}>
-    {{#if this.emptyState.link}}
-      <DocLink @path={{this.emptyState.link}}>Learn more</DocLink>
-    {{/if}}
-  </EmptyState>
-{{else}}
+{{#if (not-eq @secret.state "destroyed")}}
   <div class="info-table-row-header">
     <div class="info-table-row thead {{if this.showJsonView 'is-shadowless'}} ">
-      {{#unless this.showJsonView}}
+      {{#if (and (not this.showJsonView) (not this.emptyState))}}
```

Comment:
Could we make this another getter explicitly for showing key/value headers? It's just easier to read logic in the component instead of the template

Reply:
Definitely!